Gopkg.toml

```diff
@@ -6,6 +6,10 @@
   unused-packages = true
   non-go = true
 
+  [[prune.project]]
+    name = "gopkg.in/bblfsh/client-go.v2"
+    non-go = false
+
```

Comment:
we should add to the `.toml` the new dependencies, shouldn't we?
```go
import (
  "gopkg.in/bblfsh/client-go.v2"
  "gopkg.in/bblfsh/sdk.v1"
)
```

Reply:
I don't know. Should we?

Reply:
why not? it seems to be an app dependency, so...